pkgs/data/icons/sweet-folders/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
```

`finalAttrs` doesn't seem to be used anywhere.

```suggestion
stdenvNoCC.mkDerivation {
```